File: curso-POO/uber-project/python/car.py
from this import d
from account import Account
class Car:
    id          =int
    license     =str
    driver      =Account("","")
    passenger   =int

    def __init__(self, license, driver):
        self.driver = driver
        self.license = license
    
    def printCarData(self):
        print(f"Licencia: {self.license} --- Driver:{self.driver.name}")

File: curso-POO/uber-project/js/Car.js
function Car(licence, driver){
    this.id;
    this.licence = licence;
    this.driver = driver;
    this.passenger;
    
}

Car.prototype.printDataCar = function(){
    console.log(this.driver)
    console.log(this.driver.name)
    console.log(this.driver.document)
    console.log("Licencia: "+this.licence)
}